lbz/events/broker.py

```diff
@@ -8,17 +8,21 @@
 
 logger = get_logger(__name__)
 
+EVENT_BRIDGE_DEFAULT_TYPE_KEY = "detail-type"
+EVENT_BRIDGE_DEFAULT_DATA_KEY = "detail"
+COGNITO_DEFAULT_TYPE_KEY = "triggerSource"
+COGNITO_DEFAULT_DATA_KEY = "request"
 
-# TODO: type_key and data_key will be const for EventBridge and different set for Cognito Events
-class EventBroker(BaseHandler[None]):
+
+class BaseEventBroker(BaseHandler[None]):
```

Comment:
Once you created this class, most likely all the old tests should start using it instead 😉 

And we will need only one test for `EventBroker` exactly as you did for `CognitoEventBroker`.